spec/factories/likes.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: likes
+#
+#  id            :bigint           not null, primary key
+#  likeable_type :string           not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  likeable_id   :bigint           not null
+#  user_id       :bigint           not null
+#
+# Indexes
+#
+#  index_likes_on_likeable  (likeable_type,likeable_id)
+#  index_likes_on_user_id   (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
```

Comment:
If all these comments are not necessary you can remove them, so that the code looks cleaner.